send final result when stream is true (#87)
waylaidwanderer/node-chatgpt-api: README.md

    - using `docker-compose up` (requires Docker)

#### Usage
<details>

> <summary><strong>Method1</strong></summary>

Reviewer: ```suggestion
<summary><strong>Method 1</strong></summary>

```

    - using `docker-compose up` (requires Docker)

#### Usage
<details>

> <summary><strong>Method 1</strong></summary>

Reviewer: ```suggestion
<summary><strong>Method 1 (POST)</strong></summary>
```

    "error": "There was an error communicating with ChatGPT."
}
```
</details>
<details>

> <summary><strong>Method2</strong></summary>

Reviewer: ```suggestion
<summary><strong>Method 2</strong></summary>

```

    "error": "There was an error communicating with ChatGPT."
}
```
</details>
<details>

> <summary><strong>Method 2</strong></summary>

Reviewer: ```suggestion
<summary><strong>Method 2 (SSE)</strong></summary>
```

{ data: ' you', event: '', id: '', retry: undefined }
{ data: ' today', event: '', id: '', retry: undefined }
{ data: '?', event: '', id: '', retry: undefined }

> { data: 'Object JSON', event: 'result', id: '', retry: undefined }

Reviewer: ```suggestion
{ data: '<result JSON here>', event: 'result', id: '', retry: undefined }
```

  console.error(JSON.parse(message.data).error); // There was an error communicating with ChatGPT.
}
```
</details>

#### Notes

> - Method1 is simple but Time to First Byte (TTFB) is long.

Reviewer: ```suggestion
- Method 1 is simple, but Time to First Byte (TTFB) is long.
```

> - Method2 use non-standard implementation of [server-sent event API](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events), you should import `fetch-event-source` first and use `POST` method.

Reviewer: ```suggestion
- Method 2 uses a non-standard implementation of [server-sent event API](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events); you should import `fetch-event-source` first and use `POST` method.
```
